src/client/address_book/impl.cc

```diff
@@ -89,7 +89,7 @@ void AddressBook::Start(
     std::make_unique<boost::asio::deadline_timer>(
         m_SharedLocalDestination->GetService());
   m_SubscriberUpdateTimer->expires_from_now(
-      boost::posix_time::minutes(SubscriberTimeout::InitialUpdate));
+      boost::posix_time::minutes{static_cast<std::int64_t>(SubscriberTimeout::InitialUpdate)});
```

Comment:
and watch out for narrowing conversion warnings. Why do you need this.

Reply:
>I can replace it with the following:

Whatever you do, put an ifdef because MSVC is absolutely absurd. No one should be using microsoft to begin with.

Reply:
It is not an issue with MSVC, it is an issue with Boost 1.67.0
I can revert this change, but the problem with the latest version of Boost will remain.

Reply:
1.67 was released less than 3 days ago, after this thread started. Why were you developing on an unreleased version of boost? 1.67 hasn't even made it to mingw or arch linux yet.

Reply:
1.67 beta is available from March.
>Why were you developing on an unreleased version of boost?
Mainly because I can. As long as it does not break the build with older versions of Boost, using the newer version is not an issue.

> 1.67 hasn't even made it to mingw or arch linux yet.
It does not stop users from building it from source. And for some distros (like Ubuntu 14.04 and 16.04) it is the only possible solution to receive the recent version of Boost.

Reply:
>Mainly because I can. 

That's great. Good for you. If you want to PR to this project, you need to work *with* our build system - not against it.

>As long as it does not break the build with older versions of Boost, using the newer version is not an issue.

Adding superfluous code and overhead is an issue.

>It does not stop users from building it from source. And for some distros (like Ubuntu 14.04 and 16.04) it is the only possible solution to receive the recent version of Boost.

See note about collaboration.